app/Http/Requests/UpdateUserRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateUserRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        switch ($this->validate_rule) {
+            case config('validate.update_email'):
+                $array = ['email' => 'required|email|unique:users',];
```

Comment:
```suggestion
                $rules = ['email' => 'required|email|unique:users',];
```
đặt tên có ý nghĩa nhé